examples/gallery/database_module.py

```diff
@@ -0,0 +1,176 @@
+# -*- coding: utf-8 -*-
+"""
+=========================================
+The Database Module
+=========================================
+
+In this example we demonstrate some of the features of the Database module.
+"""
+
+##############################################################################
+# :ref:`database_guide` gives a detailed and comprehensive guide on how
+# to use the Database package.
+
+##############################################################################
+# Let's start by importing the necessary modules.
+
+from sunpy.database import Database
+from sunpy.database.tables import display_entries
+from sunpy.net import vso
+
+##############################################################################
+# First we need to instantiate a `~sunpy.database.Database` object. Note that
+# the argument can be any URL supported by SQLAlchemy.
+
+database = Database('sqlite:///sunpydata.sqlite')
+
+##############################################################################
+# Adding Entries from a VSO query
+
+vso_query_result = vso.VSOClient().query(
+	vso.attrs.Time('2011-05-08', '2011-05-08 00:00:05'),
+	vso.attrs.Instrument('AIA'))
+database.add_from_vso_query_result(vso_query_result)
+
+##############################################################################
+# Now to see what is in the database, we use the display_entries function to
+# display the desired parameters in a neat way.
+
+print(display_entries(database,
```

Comment:
I think `display_entries` should have a sane set of deaults, it seems stupid to specify this list everytime when 90% of the time you want the same columns.